integrations/pr_publish.py

```
@@ -0,0 +1,902 @@
+            if not isinstance(c, dict):
+                continue
+            body = c.get("body") or ""
+            author = (c.get("user") or {}).get("login") if isinstance(c.get("user"), dict) else None
+            cid = c.get("id")
+            if (MANAGED in body and VERDICT_MARKER in body and author == owner_login
+                    and isinstance(cid, int) and cid != bootstrap_cid):
+                owned.append(cid)
+        if owned:
+            owned.sort()  # GitHub comment ids increase monotonically → the smallest is the oldest
+            summary_cid = owned[0]
+            for cid in owned[1:] + [bootstrap_cid]:  # retire extras + the throwaway bootstrap comment
```

Ruff (0.16.1)

[warning] Consider `[*owned[1:], bootstrap_cid]` instead of concatenation

Replace with `[*owned[1:], bootstrap_cid]`

(RUF005)